### PR TITLE
update ACT quick start demo

### PR DESCRIPTION
**Pull request by yghstill**

### CODE REVIEW

File: demo/auto_compression/README.md [hunk 1]
```diff
@@ -14,20 +14,46 @@ PaddleSlim推出全新自动压缩工具（ACT），旨在通过Source-Free的
...
+train_loader = paddle.io.DataLoader(
+    train_dataset,
+    feed_list=[image],
+    drop_last=False,
+    return_list=False,
+    batch_size=32,
+    shuffle=False)
+# 加载配置文件
+compress_config, train_config = load_config(
+    "./image_classification/configs/mobilenetv1_qat_dis.yaml")
```
Comment (qingqing01) [inline demo/auto_compression/README.md]: 这个config里的内容是啥？能否不设置？

File: demo/auto_compression/README.md [hunk 2]
```diff
@@ -14,20 +14,46 @@ PaddleSlim推出全新自动压缩工具（ACT），旨在通过Source-Free的
...
 from paddleslim.auto_compression.config_helpers import load_config
 from paddleslim.auto_compression import AutoCompression
-from paddleslim.common.imagenet_reader import reader
-# 加载配置文件
-compress_config, train_config = load_slim_config("./image_classification/mobilenetv1_qat_dis.yaml")
+from paddleslim.common.imagenet_dataset import ImageNetDataset
+paddle.enable_static()
 # 定义DataLoader
-train_loader = reader(mode='test') # DataLoader
+train_dataset = ImageNetDataset(
```
Comment (qingqing01) [inline demo/auto_compression/README.md]: paddle.vision下面的 DatasetFolder等不能用吗？参考 https://github.com/PaddlePaddle/hapi/blob/master/image_classification/imagenet_dataset.py

Comment (yghstill) [reply demo/auto_compression/README.md]: 不能直接用，hapi中dataset返回的是img, label，我们只需要img，否则dataloader需要再包装一层

Comment (qingqing01) [reply demo/auto_compression/README.md]: 这个呢 https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/vision/datasets/ImageFolder_cn.html

这里简单示例，想展示的全流程，每个流程需要啥，少量代码即可运行。
现在Dataset和configs/mobilenetv1_qat_dis.yaml 都太黑盒

Comment (yghstill) [reply demo/auto_compression/README.md]: ImageNet dataset是通过train_list.txt和val_list.txt解析路径及class然后读取数据，ImageFolder这个是每个文件夹存储一个类别的数据，和标准ImageNet数据解析流程不太一样，无法完全复用。

File: demo/auto_compression/README.md (hunk 1 above)
Comment (yghstill) [reply demo/auto_compression/README.md]: done

File: demo/auto_compression/README.md (hunk 2 above)
Comment (yghstill) [reply demo/auto_compression/README.md]: DONE

**Review by ceci3 — APPROVED** [review]
